lib/timers.js

```diff
@@ -51,7 +51,11 @@ const { kInit, kDestroy, kAsyncIdCounter } = async_wrap.constants;
 const async_id_symbol = timerInternals.async_id_symbol;
 const trigger_async_id_symbol = timerInternals.trigger_async_id_symbol;
 
-const [activateImmediateCheck, scheduledImmediateCountArray] =
+// should match ImmediateInfo in src/env.h
+const kCount = 0;
+const kOutstanding = 1;
```

Comment:
It’s not really obvious that this is a 0/1 flag, so I’d prefer to have it called something like `kHasOutstanding`?